Assets/Mirror/Runtime/Transport/Transport.cs

```diff
@@ -24,6 +16,31 @@ public abstract class Transport : MonoBehaviour
         /// </summary>
         public static Transport activeTransport;
 
+        public Transport()
+        {
+            // use contructor instead of awake
+            // - other transports dont need to override
+            // - setting actions here is safe because it doesn't effect unity things
+            ResetClientHandlers();
+            ResetServerHandlers();
+        }
+
+        public void ResetClientHandlers()
+        {
+            OnClientConnected = () => Debug.LogWarning("onClientConnected called with no handler");
```

Comment:
please do this where OnClientConnected etc. are defined.
it looks like you exposed ResetHandlers for NetworkClient, but we don't actually need that function anymore, see other comment in Networkclient